### PR TITLE
Extend the bibliography

### PR DESCRIPTION
**Pull request by wilfwilson**
I intend for this to address, at least to some degree, #78.

It seems that GAPDoc chooses its own keys for the bib entries when compiling the documentation (e.g. `JLNP13`), no matter which key you supply in the bib file. So I have just used the GAPDoc-generated keys for each entry, so that everything is consistent.

I have currently cited all of the new references in a temporary section in the manual, so that all entries are compiled into the bibliography at the end of the manual. This is perhaps controversial.

### CODE REVIEW

File: doc/intro.xml [hunk 1]
```diff
@@ -18,6 +18,11 @@ in this package is useful at least for permutation groups, matrix
 groups and projective groups.
 The setup is described in <Cite Key="NS06"/>. <P/>
 
+<Cite Key="LNPS06"/>
+<Cite Key="BLS97"/>
+<Cite Key="JLNP13"/>
+<Cite Key="KS09"/>
```
Comment (wilfwilson) [inline doc/intro.xml]: These are currently just here so that they appear in the bibliography in the compiled doc.

Comment (fingolfin) [reply doc/intro.xml]: Makes sense

**Comment by wilfwilson** [comment]
I'm still looking to see if there is any low-hanging 'fruit' (references) in the package for me to add to the bibliography. I think there's just one or two more (currently adding Max's habilitation thesis).

Then I have to decide whether to find the appropriate places in the manual to cite each of these items in this PR. I'm thinking perhaps that should be a separate PR, and for now I can just cite everything in a separate place, or not at all. That way the items can be in the bib file, for people to use if they're also writing documentation (as e.g. @ssiccha is).

**Comment by wilfwilson** [comment]
@fingolfin I've done what I'll do for now. If you're unhappy with the temporary literature section, I can simply remove it. Otherwise I'd be happy for this to be merged.

**Pull request merged**